ppx/ppx_js/lib_internal/ppx_js_internal.ml

```diff
@@ -451,6 +471,7 @@ let new_object constr args =
     invoker
     ((app_arg (Exp.ident ~loc:constr.loc constr) :: args)
     @ [ app_arg (unit ~loc:gloc ()) ])
+  [@@ocaml.warning "-26"]
```

Comment:
what is this warning 26 about ?

Reply:
I don't need to disable that warning anymore. I needed that when ppxlib was still dependent on Base. I've gotten rid of disabling it. Thanks.